### DISCUSSION TITLE
Should the app name string be translatable?

### DISCUSSION THREAD

**Discussion by bgo-eiu** [streetcomplete/StreetComplete#4479] [Ideas]
Some applications on Android show a localised name in the launcher; others use a localised label. On one hand, some consistency on how an app is named can be desirable, but on the other it can look out of place when mixing scripts.

I am ask partly because in translating the app to Punjabi, I came up with a name that is rhyming and more idiomatic: "galli poori" (گلی پوری). This literally just translates to "street complete" but I thought that would be kind of cute and easier to pronounce

**Comment by mcliquid** [comment]
Just out of curiosity: Are big player apps like Facebook, WhatsApp, PayPal,... and so on also translated?

**Reply by bgo-eiu** [reply]
The Microsoft "Phone Link" app that shows texts on the computer and the Wikipedia + Wikimedia Commons apps are the "big players" I have installed which show translated names in the launcher. (The rest that do this are open source and/or not available in English to begin with.) Spotify, Instagram, and Whatsapp are the big players I have on my phone which have support the locale on my phone in-app but which do not have translated names.

The Android default apps show up in a mix of languages, their localizations are outdated and getting them fixed is slow and laborious

**Comment by riQQ** [comment]
As the name StreetComplete is more similar to app names like WhatsApp or Instagram and not like Calculator or Calendar, I wouldn't translate it.
